addons/hr_calendar/tests/test_working_hours.py

```diff
@@ -0,0 +1,236 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged
+from datetime import datetime
+from odoo.addons.hr_calendar.tests.common import TestHrCalendarCommon
+
+
+@tagged('work_hours')
+class TestWorkingHours(TestHrCalendarCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        if 'hr.contract' in cls.env:
+            cls.skipTest(cls,
+            "hr_contract module is installed. To test these features you need to install test_hr_contract_calendar")
```

Comment:
exactly 120 :scream: 
```suggestion
            cls.skipTest(cls,
                "hr_contract module is installed. To test these features you need to install test_hr_contract_calendar")
```

Reply:
Perfection :ok: